app/assets/v2/css/quests.css

```diff
@@ -1400,3 +1400,73 @@ body.green .tl{
   }
 }
 
+/* Responsive for iPhone 6/7/8/Plus & X */
+
+@media (max-width: 500px) {
```

Comment:
Can we change this to @media (max-width: 575.98px) { ... } ? 
This is what bootstrap responsive breakpoints which is usually what we end up following 

https://getbootstrap.com/docs/4.3/layout/overview/

Reply:
Added.